client/apps/user-portal/src/app/router.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { SermonsComponent, SermonListComponent } from '@app/sermons';
+
+const routes: Routes = [
+  { 
+    path: '', component: SermonsComponent, children: [
```

Comment:
So, I think the components are a little off.

The way the `users` app works is that the main route will have a list of sermons for people to scroll through.
Once a user selects a video, the should be routed to a view of that specific video they selected.

Reply:
So the main route is to, the sermons component, but some sort of 'sermons-view' should be available on the '/:id' route. Correct?

Reply:
Would the sermon-view component be usable in the admin app as well as the users?

Reply:
As of now, no, the admin would not see the `sermon-view` component.